core/src/test/java/org/apache/calcite/test/SqlFunctionsTest.java

```diff
@@ -254,18 +254,88 @@ static <E> List<E> list() {
       final boolean b = f.regexpContains("abc def ghi", "[z-a]");
       fail("expected error, got " + b);
     } catch (RuntimeException e) {
-      assertThat(e.getMessage(),
-          is("Invalid regular expression for REGEXP_CONTAINS: 'Illegal "
-              + "character range near index" + " 3 [z-a]    ^'"));
+      assertThat(
+          e.getMessage(), is("Invalid regular expression for REGEXP_CONTAINS: 'Illegal "
```

Comment:
Is this formatting right?

Reply:
It's not throwing any formatting errors, I just removed the extra str concat within the same line as it looked unnecessary